protos/core.proto

```diff
@@ -26,22 +26,62 @@ message ProjectConfig {
   string gcloud_project_id = 6;
 }
 
-message Environment {
-  string name = 1;
+message Environments {
+  message Environment {
+    string name = 1;
+    
+    message GitReference {
+      oneof git_reference {
+        string branch = 1;
+        string commit_sha = 2;
+      }
+    }
+  
+    GitReference git_reference = 2;
+    ProjectConfig config_override = 3;
+
+    message Schedule {
+      // Required fields.
+      string name = 1;
+      string cron = 2;
+      repeated string tags = 3;
   
-  message GitReference {
-    oneof git_reference {
-      string branch = 1;
-      string commit_sha = 2;
+      // Optional.
+      bool include_dependencies = 4;
```

Comment:
Why not put this stuff in a new type?
It's basically a stripped down RunConfig, I understand why we don't use that directly here but will be easier to keep in sync if we have seperate messages

Reply:
https://github.com/dataform-co/dataform/pull/722